stories/Folder-Structure.stories.mdx

```diff
@@ -0,0 +1,44 @@
+import { Meta } from '@storybook/addon-docs';
+import { H1 } from '@leafygreen-ui/typography';
+import BannerImage from '../.storybook/static/leafygreen-ui-banner.png';
+
+<Meta title="Developer Guide/Package Folder Structure" />
+
+### Folder Structure
+
+- `index.tsx`
+- `styles.ts`
+- `types.ts`
```

Comment:
For the same reason we keep the component in `<ComponentName>.tsx` not index, I would advocate for using `<ComponentName>.styles.tsx`

Reply:
Additionally, this keeps names consistent with `*.stories.tsx` and `*.spec.tsx` files